kalite/distributed/static/js/distributed/topics/router.js

```diff
@@ -1,6 +1,8 @@
-ChannelRouter = Backbone.Router.extend({
+TopicChannelRouter = Backbone.Router.extend({
     initialize: function(options) {
+        _.bindAll(this);
         this.default_channel = options.default_channel;
+        $("#nav_learn").click(this.intercept_learn_nav);
```

Comment:
I wonder whether we could handle this in a more general way (perhaps not now, to keep risk low, but as a TODO in there). It could work like this:

Intercept any click event that bubbles up to `$('body')`, and check whether the `href` begins with the root on the current base Backbone router. If so, route through there and return `false`. Otherwise, let it carry on with a page nav.